src/Tasks/Common/Resources/Strings.resx

```diff
@@ -896,4 +896,13 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1198: A publish profile with the name '{0}' was not found in the project. Set the PublishProfile property to a valid file name.</value>
     <comment>{StrBegin="NETSDK1198: "}</comment>
   </data>
+  <data name="ArtifactsPathCannotBeSetInProject" xml:space="preserve">
+    <value>NETSDK1199: The ArtifactsPath and UseArtifactsOutput properties cannot be set in a project file, due to MSBuild ordering constraints. They must be set in a Directory.Build.props file or from the command line. See https://aka.ms/netsdk1199 for more information.</value>
```

Comment:
Do we need to manually lock the ArtifactsPath and UseArtifactsOutput strings inside a `<note />` section for the translators so they cannot translate the property names by accident?

Reply:
Might be a good idea, can you point me to an example of what that looks like.

Reply:
You can add something like:
`<note>{Locked="dotnet workload search"}</note>`
if `dotnet workload search` is what you want to lock.

This PR has a lot of examples: https://github.com/dotnet/sdk/pull/28553/files

Reply:
In the ENU resx file you should be able to add something like
```xml
<comment>{Locked="ArtifactsPath, UseArtifactsOutput"}</comment>
```
The comments will automatically be added in the XLF as notes for the translators IIRC